src/OAuth2Demo/Client/Controllers/RequestResource.php

```diff
@@ -34,8 +34,6 @@ public function requestResource(Application $app)
         $response = $request->send();
         $json = json_decode((string) $response->getBody(), true);
 
-        $resource_uri = sprintf('%s%saccess_token=%s', $endpoint, false === strpos($endpoint, '?') ? '?' : '&', $token);
```

Comment:
The link at the bottom of the show resource page still does not have an access token, and that is what I'm referring to. 

If you can revert this line I'll merge the PR.